.gitignore

```diff
@@ -7,4 +7,5 @@ coverage/
 !univ3-lps.csv
 !badges-for-bandits-nfts.csv
 !master-list.csv
+!distribution.csv
```

Comment:
did you want to check in this file too?

Reply:
Will add this file after we swap arb to eth